airflow/providers/amazon/aws/utils/emailer.py

```diff
@@ -31,7 +31,7 @@ def send_email(
     bcc: Optional[Union[List[str], str]] = None,
     mime_subtype: str = 'mixed',
     mime_charset: str = 'utf-8',
-    conn_id: Optional[str] = None,
+    conn_id: str = 'ses_default',
```

Comment:
All AWS integration should share common conn_id. See: https://github.com/apache/airflow/blob/master/UPDATING.md#normalize-gcp_conn_id-for-google-cloud
If the user needs it, he can define more complex configurations, but the default should assume that all AWS / GCP services have one credential.